NextcloudTalk/SettingsTableViewController.swift

```diff
@@ -288,13 +290,23 @@ class SettingsTableViewController: UITableViewController, UITextFieldDelegate {
         self.navigationController?.pushViewController(userProfileVC, animated: true)
     }
 
-    // MARK: User Status
+    // MARK: User Status (obsolete)
 
-    func presentUserStatusOptions() {
+    /*func presentUserStatusOptions() {
         if let activeUserStatus = activeUserStatus {
             let viewController = UserStatusTableViewController(userStatus: activeUserStatus)
             self.navigationController?.pushViewController(viewController, animated: true)
         }
+    }*/
```

Comment:
If we don't need it, we can remove it (also the file, if the file still exists?)